docs/_blog/liana-payjoin.md

```diff
@@ -0,0 +1,45 @@
+date: "2025-04-10"
+authors:
+  - Armin Sabouri
+tags:
+  - privacy
+  - integrations
+  - cut-through
+---
+
+[Liana](https://wizardsardine.com/liana/) is a Bitcoin desktop wallet that enables users to set up inheritance policies, ensuring their funds can be passed on to loved ones only after a predefined delay called a "timelock". To prevent unintentional activation of the inheritance policy, these time locked coins require periodic refreshing. Liana's Payjoin integration enables users to refresh timelocks as they receive payments, boosting privacy and cutting costs.
+
+## Privacy Leak Explained
+
+Inheritance planning is a crucial yet often overlooked part of Bitcoin self-custody. While Bitcoin offers unmatched financial sovereignty, it introduces [unique challenges](https://wizardsardine.com/blog/the-state-of-bitcoin-inheritance-planning/) when it comes to passing on funds to future generations. Liana addresses this with inheritance policies that activate only after a timelock. For example, a user might control a primary key for daily use, while a recovery key held by a beneficiary is time locked for future access.
+
+However, there's a catch: most users do not frequently spend their Bitcoin. A [2024 study](https://www.sciencedirect.com/science/article/pii/S0378437124008045) found that the age of unspent coins follows a power-law distribution (see Figure 3 that study). In practice, this means coins are typically either spent soon after receipt or held for long periods. [Data also suggests](https://charts.bitbo.io/hodl-waves/) that most coins remain unspent for months or years. To avoid unintentionally triggering a recovery path, users often resort to reconciliation of their coins to reset timelocks.
+
+<img alt="Liana time locked coins" src="../assets/liana-coins.png" />
+
+But this reconciliation comes at a cost to privacy. These transactions create an identifiable on-chain footprint, making it easier for on-chain surveillance to estimate a user's balance and transaction history. This is primarily due to the [common input ownership heuristic](https://en.bitcoin.it/wiki/Common-input-ownership_heuristic), which assumes all inputs in a transaction belong to the same entity. Combined with unique transaction patterns that act like fingerprints, these heuristics allow on-chain analysts to cluster a user's addresses and glean sensitive information.
+
+## Payjoin as a Privacy-Preserving Refresh
+
+This is where Payjoin shines. It allows Liana users to refresh near-expiry coins during normal payments. For example, when a user receives a payment via Payjoin, they can opportunistically include expiring coins to refresh their timelocks. This same logic can apply to self-spends&mdash;users can initiate a Payjoin when sending a transaction, embedding refresh into routine activity.
+
+Whether as sender or receiver, users benefit from lower fees, improved privacy, and preservation of their inheritance policies.
```

Comment:
"Whether _ as a sender or receiver" is missing a verb.

---

is preservation the correct word choice in this sentence? technically the policy is *maintained* by being renewed, because preserving it would allow it to expire. I prefer "maintenance" or "renewal"

Reply:
Ended up removing this sentence. Unnecessary transtion imo